src/Transport.ts

```diff
@@ -74,14 +75,21 @@ export class Transport {
     private _remoteHandlerDeletionCallbacks:
         { [slotName: string]: RemoteHandlerCallback } = {}
 
+    /**
+     * Callbacks provided by each slot allowing to remove blacklisted events
+     * declaration from the remote handlers.
```

Comment:
The use cases that it can handle is not tied to the functionality this implementation provides. I wouldn't list what can be done with it as that can change and is determined elsewhere. We should only comment about what this implementation specifically does.